server/s3_delete_multiple_objects_action.cc

```diff
@@ -342,6 +345,41 @@ void S3DeleteMultipleObjectsAction::fetch_objects_extended_info_failed() {
   s3_log(S3_LOG_DEBUG, "", "%s Exit", __func__);
 }
 
+void S3DeleteMultipleObjectsAction::save_bucket_counters() {
+  s3_log(S3_LOG_INFO, stripped_request_id, "%s Entry\n", __func__);
+  int64_t inc_object_count = 0;
+  int64_t inc_obj_size = 0;
+
+  inc_object_count = -objects_metadata.size();
+  for (auto& object_metadata : objects_metadata) {
+    inc_obj_size -= (object_metadata->get_content_length());
+  }
```

Comment:
Should we maybe do this loop similar to `delete_objects_metadata()`?  I.e. check that object metadata is valid?  As only those objects are actually deleted in delete_metadata call.

```
  for (auto& obj : objects_metadata) {
    if (obj->get_state() != S3ObjectMetadataState::invalid) {
      keys.push_back(obj->get_object_name());
    }
  }
```

Reply:
Done